src/app/components/media/CreateRelatedMedia.js

```diff
@@ -39,6 +39,20 @@ class CreateRelatedMedia extends Component {
     this.setState({ dialogOpen: false });
   };
 
+  currentProject() {
+    let project = null;
+    let currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
+    if (currentProjectId) {
+      currentProjectId = parseInt(currentProjectId[1], 10);
+      project = this.props.media.projects.edges.filter(p =>
+        parseInt(p.node.dbid, 10) === currentProjectId);
+      if (project.length) {
+        project = project[0].node;
+      }
+    }
```

Comment:
I recommend using `find` instead of `filter` for a more readable, more idiomatic code.
E.g.: ```return projects.edges.find(p => `${p.node.dbid}` === currentProjectId)```

Reply:
Done